core/src/test/java/com/yahoo/oak/KeyBufferTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class KeyBufferTest {
+
+    private SliceUtils sliceUtils;
+
+    @Before
+    public void setUp() throws Exception {
+        sliceUtils = new SliceUtils();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        sliceUtils.close();
+    }
+
+    @Test
+    public void byteBufferShouldBeReadOnly() {

```

Comment:
Can you merge all the test classes you added into one test class? They seem to fit the same pattern and they test similar things. It makes sense to group them together.

Reply:
@liran-funaro The reason I separated them that way was that it's a test among many others that may be added in the future.  If you still think it's better to consolidate them all, then I'll add another commit for doing so. Please give your final answer here. Thanks.

Reply:
Thanks for the explanation. Since we don't have other tests for these buffers yet, I think it is better to consolidate them into a single class. But I do agree each test should be in its own method.

Reply:
Great. Thank you for your PR.

Reply:
I pushed another commit to fulfill the request from you. Thank you.